SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -865,7 +865,7 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	if (IsServer())
+	if (IsServer() && WellKnownEntitySystem != nullptr)
```

Comment:
IsValid() would have been more consistent with similar usage elsewhere in this file?

Reply:
I see both being used interchangeably maybe something for a styleguide discussion